File: INICIANTE/JAVASCRIPT/1044.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var [a, b] = prompt("pontos").split(" ").map(Number)

if( a % b == 0){
    console.log("Sao Multiplos")
}
else if(b % a == 0){
    console.log("Sao Multiplos")
}
else{
    console.log("Nao sao Multiplos")
}

File: INICIANTE/JAVA/1044.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int a, b;
        Scanner sc = new Scanner(System.in);
        a = sc.nextInt();
        b = sc.nextInt();

        if( a % b == 0){
            System.out.println("Sao Multiplos");
        }
        else if(b % a == 0){
            System.out.println("Sao Multiplos");
        }
        else{
            System.out.println("Nao sao Multiplos");
        }
        sc.close();
    }
}
